packages/common/src/types/catalog.ts

```diff
@@ -57,6 +57,7 @@ export type CatalogMetadata = {
     modelName: string;
     source: string | undefined;
     fields: CatalogField[];
+    joinedTables: string[];
```

Comment:
I thought we already had a property like this, to display joins in the list of tables. Right @magnew ?

Reply:
That's a great point @rephus, I checked that code but made more sense (to me) to add this to the response of the `metadata` - since it's a _simple_ map over the explore. 

I'm going ahead and merge this. I feel like this case is a great opportunity to make more refactors/improvements later 😄